muse/CONST.py

```diff
@@ -7,7 +7,7 @@
 KEEP_ALIVE_TIMEOUT = os.environ.get("KEEP_ALIVE_TIMEOUT", 160)
 RATE_LIMIT_KEY = os.environ.get("RATE_LIMIT_KEY", str(uuid.uuid4().hex))
 SENTRY_API_KEY = os.environ.get("SENTRY_API_KEY", None)
-MUSE_SYSTEM_PASSWORD = os.environ.get("MUSE_SYSTEM_PASSWORD", "").encode("utf-8")
+MUSE_SYSTEM_PASSWORD = os.environ.get("MUSE_SYSTEM_PASSWORD", "")
```

Comment:
I think this was added on purpose to parse some special characters which may appear in the password...

Reply:
we do it later in the code
```
secrets.compare_digest(current_password_bytes, MUSE_SYSTEM_PASSWORD.encode("utf8"))
```